.github/workflows/nightly.yml

```diff
@@ -117,10 +117,95 @@ jobs:
         run: |
           cd "${BUILD_DIR}"
           cmake --build "${BUILD_DIR}" --target package
+      - name: Get version info
+        run: |      
+          cd ${{ env.SRC_DIR }}
+          echo "VERSION=$(git describe --tags | sed 's/^release-//;s/-/+/;s/-/~/;s/rc/~rc/')-$(arch)" >> $GITHUB_ENV
```

Comment:
Good idea, but I have some doubts about using `arch` (or `uname -m`) command on MSYS2: I _think_ it'll always return `x86_64` because MSYS2 is currently only built for that (MSYS2 being the bash shell running here and the POSIX emulation layer and tools, not the native build UCRT64 toolchain environment...), and I don't think that'll change for while. However, there is also CLANGARM64 available where darktable already builds, so we could potentially offer it in the near future, and `arch` wouldn't match in that case.

I suggest we try to use the lowercased `${PROCESSOR_ARCHITECTURE,,}` Windows environment variable (should give us "amd64" or "arm64"), or ${MINGW_CARCH} ("x86_64" or "aarch64"), or something similar...

Reply:
ok,

    echo "VERSION=$(git describe --tags | sed 's/^release-//;s/-/+/;s/-/~/;s/rc/~rc/')-${PROCESSOR_ARCHITECTURE,,}"

now results in

    4.3.0+778~gf1ef7cb62-amd64

Looks good?

Reply:
Better! But I'm 50/50 on "amd64", it's a bit unusual although that is the official Microsoft target, but most docs & people don't really use it... Let's just go w/ `${MSYSTEM_CARCH}` for now?

Reply:
👍done